firedrake/preconditioners/__init__.py

```diff
@@ -0,0 +1,5 @@
+from firedrake.preconditioners.base import *
+from firedrake.preconditioners.assembled import *
+from firedrake.preconditioners.massinv import *
+from firedrake.preconditioners.pcd import *
+from firedrake.preconditioners.patch import *
```

Comment:
This is just code movement to rationalise the increasing number of preconditioners.